src/services/link/index.js

```diff
@@ -16,29 +17,52 @@ module.exports = function setup() {
 		lean: true,
 	};
 
+	const registrationLinkExpirationDays = REGISTRATIONLINK_EXPIRATION_DAYS || 30;
+
 	let linkService = service(options);
 
-	function redirectToTarget(req, res, next) {
-		if (req.method === 'GET' && !req.query.target) { // capture these requests and issue a redirect
+	const isExpired = (date) => {
+		// calculate the date the link will expire on
+		date.setDate(date.getDate() + registrationLinkExpirationDays);
+		// check if that expiration date has already passed
+		return date <= Date.now();
+	};
+
+	const isLocalRegistrationLink = (url) => {
+		// calculate the beginning of a registrationLink e.g. https://schul-cloud.org/registration/
+		const linkPrefix = `${app.settings.services.web}/registration/`;
+		// check if url starts with that prefix
+		return (url.startsWith(linkPrefix));
+	};
+
+	function checkLink(req, res, next) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected to return a value at the end of function 'checkLink'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=3811219)